test/uk/gov/hmrc/tai/util/viewHelpers/JsoupMatchers.scala

```diff
@@ -236,6 +236,11 @@ trait JsoupMatchers {
   def havePreHeadingWithText(expectedText: String, expectedPreHeadingAnnouncement: String = "This section is") =
     new CssSelectorWithTextMatcher(s"$expectedPreHeadingAnnouncement $expectedText", "header>p")
 
+  def havePreHeadingWithTextNewTemplate(
+    expectedText: String,
+    expectedPreHeadingAnnouncement: String = "This section is") =
+    new CssSelectorWithTextMatcher(s"$expectedPreHeadingAnnouncement $expectedText", "span")
+
```

Comment:
should this defo be a span?

Reply:
@masahmad ?

Reply:
![sgScreenshot 2022-05-04 151956](https://user-images.githubusercontent.com/6519788/166701563-821e09cf-b126-4d7e-a26e-dbc456cb4e48.png)

Think its fine -  as the  header does have a span ( see screenshot)  - we have 2 implementations  in ptaHeader hence why we pass in GDSVersion=true param to use the appropriate one ( new pages use gds version which uses span )